components/omega/doc/userGuide/MetaData.md

```diff
@@ -0,0 +1,245 @@
+(omega-user-metadata)=
+
+# What is Metadata in Omega?
```

Comment:
These headers are level 1 headers and need to be subheads under a main MetaData header at the top.